pkg/secretsencrypt/controller.go

```diff
@@ -89,8 +90,17 @@ func (h *handler) onChangeNode(key string, node *corev1.Node) (*corev1.Node, err
 		return node, err
 	}
 	ann = EncryptionReencryptActive + "-" + reencryptHash
-	node.Annotations[EncryptionHashAnnotation] = ann
-	node, err = h.nodes.Update(node)
+
+	nodeName := node.Name
+	err = retry.RetryOnConflict(retry.DefaultRetry, func() error {
+		node, err := h.nodes.Get(nodeName, metav1.GetOptions{})
```

Comment:
nit - is this worth a variable, or can we just use the name of the original node object? `key` is also the node name, if that helps.
```suggestion
	err = retry.RetryOnConflict(retry.DefaultRetry, func() error {
		node, err := h.nodes.Get(node.Name, metav1.GetOptions{})
```

Reply:
Did not know key was the nodeName, fixed